src/material-experimental/mdc-button/_mdc-button.scss

```diff
@@ -1,9 +1,23 @@
 @import '@material/button/mixins';
+@import '@material/button/variables';
 @import '@material/fab/mixins';
 @import '@material/ripple/mixins';
 @import '@material/icon-button/mixins';
 @import '../mdc-helpers/mdc-helpers';
 
+// Applies the disabled theme color to the text color.
+@mixin _mat-button-disabled-color() {
+  @include mdc-theme-prop(color,
+      mdc-theme-ink-color-for-fill_(disabled, $mdc-theme-background));
+}
+
+// Applies the disabled theme background color for raised buttons. Value is taken from
+// mixin `mdc-button--filled`
+@mixin _mat-button-disabled-background() {
+  @include mdc-theme-prop(background-color, rgba(mdc-theme-prop-value(on-surface), 0.12));
```

Comment:
The opacity here should come from variables. E.g. `map-get($mdc-ripple-dark-ink-opacities, hover)`, although I'm unsure what the correct variable name is. You can check the MDC-based slide toggle for reference.

Reply:
Add a TODO?